src/app/api/crawl/crawler.ts

```diff
@@ -80,8 +102,20 @@ class Crawler {
 
   private extractUrls(html: string, baseUrl: string): string[] {
     const $ = cheerio.load(html);
```

Comment:
Minor observation: This will give a 'deprecated warning. You can do `import { load } from 'cheerio';` and use the load by itself instead.